### PR TITLE
add BLK-BEP20, set LUNA and UST wallet_only, ws_url for couple coins

### PR DESCRIPTION
**Pull request by cipig**
https://github.com/KomodoPlatform/coins/pull/468

### CODE REVIEW

**Comment by smk762** [comment]
Awaiting second review approval and merge of https://github.com/KomodoPlatform/coins/pull/468

**Review by didwhat — DISMISSED** [review]

**Review by smk762 — APPROVED** [review]

**Review by SylEze — APPROVED** [review]

**Pull request merged**
